CIP-1694/README.md

```diff
@@ -0,0 +1,1124 @@
+All submitted governance actions will therefore either:
+
+1. be **ratified**, then **enacted**
+2. be **ratified**, yet **dropped** as a result of some higher priority action
+3. **expire** after a number of epochs
+
+Deposits are returned immediately when either:
+
+1. a ratified action is **enacted**
+2. the action **expires**
+3. a ratified action is **dropped**
+
+Some actions will be enacted _immediately_ (i.e. at the same epoch boundary they are ratified), others are enacted only on _the following epoch boundary_.
+
+| Governance action type         | Enactment           |
+| :--                            | :--                 |
+| 1. Motion of no-confidence     | Immediate           |
+| 2. New committee/quorum        | Immediate           |
+| 3. Update to the Constitution  | Immediate           |
+| 4. Hard-fork initiation        | Next epoch boundary |
+| 5. Protocol parameter changes  | Next epoch boundary |
+| 6. Treasury withdrawal         | Immediate           |
+| 7. Info                        | Immediate           |
+
+<!-- TODO - break up the protocol parameters into those which effect the header/body validation split (so that some can be enacted immediately)? -->
+
+#### Content
+
+Every governance action will include the following:
+
+* a deposit amount (recorded since the amount of the deposit is an updatable protocol parameter)
+* a reward address to receive the deposit when it is repaid
+* an anchor for any metadata that is needed to justify the action
+* a hash digest value to prevent collisions with competing actions of the same type (as described earlier)
+
+<!-- TODO: Provide a CBOR specification in the annexe for these new on-chain entities -->
+
+In addition, each action will include some elements that are specific to its type:
+
+| Governance action type         | Additional data                                               |
+| :--                            | :--                                                           |
+| 1. Motion of no-confidence     | None                                                          |
+| 2. New committee/quorum        | The set of verification key hash digests and a quorum number  |
+| 3. Update to the Constitution  | A hash digest of the Constitution document                    |
+| 4. Hard-fork initiation        | The new (greater) major protocol version                      |
+| 5. Protocol parameters changes | The changed parameters                                        |
+| 6. Treasury withdrawal         | A map from stake credentials to a positive number of Lovelace |
+| 7. Info                        |                                                               |
+
+Each accepted governance action will be assigned a unique identifier (a.k.a. the **governance action ID**),
+consisting of the transaction hash that created it and the index within the transaction body that points to it.
+
+#### Protocol Parameter groups
+
+We have grouped the protocol parameter changes.
```

Comment:
I don't think that splitting up the parameters is necessarily a good idea. In some scenarios, it may be necessary to adjust multiple parameters across various groups to maintain balance within the system.
Governance actions should provide sufficient details on the rationale behind the parameter changes. DReps should decide when to abstain on a per-action basis, rather than by parameter group.

Reply:
The groups are about different difficulties of changing the parameters, not about preventing different things to be changed at the same time. We should probably add a sentence saying that it is possible to change parameters from different groups at the same time, and the threshold is the maximum of all the groups involved. What do you think @JaredCorduan ?

Reply:
yes, absolutely, sorry that was so unclear. what @WhatisRT described is how I had intended it.